pkg/validation/validate.go

```diff
@@ -43,14 +43,34 @@ const (
 	DuplicateLabelNamesErrorMsg = "stream '%s' has duplicate label name: '%s'"
 )
 
+// TruncatedLines is a metric of the total number of lines mutated, by reason.
+var TruncatedLines = prometheus.NewCounterVec(
```

Comment:
`<nit>`Can we rename this to match the metric itself? `TruncatedLines` seems out of sync now with `mutated_samples_total`, and ditto for the other one.`</nit>`